website/docs/guides/container_service_extension_4_0.html.markdown

```diff
@@ -0,0 +1,411 @@
+  If you are going to use more than one [Edge Cluster][edge_cluster], please consider modifying the proposed configuration.
+  In UI, [Edge Clusters][edge_cluster] can be found in the NSX-T manager web UI.
+- `network_pool_name`: This references an existing Network pool and it's used to create both VDCs.
+  If you are going to use more than one Network pool, please consider modifying the proposed configuration.
+
+In the [proposed configuration][step2] the Tenant Organization's VDC has all the required VM Sizing Policies assigned, with the `TKG small` being the default one.
+You can customise it to make any other TKG policy the default one.
+
+You can also leverage changing the storage profiles and other parameters to fit the requirements of your organization. Also,
+if you already have usable [VDCs][vdc], you can change the configuration to fetch them instead.
+
+#### Catalog and OVAs
+
+The [proposed configuration][step2] will create two catalogs:
+
+- A catalog to host CSE Server OVA files, only accessible to CSE Administrators. This catalog will allow CSE Administrators to organise and manage
+  all the CSE Server OVAs that are required to run and upgrade the CSE Server.
+- A catalog to host TKGm OVA files, only accessible to CSE Administrators but shared as read-only to tenants, that can use them to create TKGm clusters.
+
+Then it will upload the required OVAs to them. The OVAs can be specified in `terraform.tfvars`:
+
+- `tkgm_ova_folder`: This will reference the path to the TKGm OVA, as an absolute or relative path. It should **not** end with a trailing `/`.
+- `tkgm_ova_file`: This will reference the file name of the TKGm OVA, like `ubuntu-2004-kube-v1.22.9+vmware.1-tkg.1-2182cbabee08edf480ee9bc5866d6933.ova`.
+- `cse_ova_folder`: This will reference the path to the CSE OVA, as an absolute or relative path. It should **not** end with a trailing `/`.
+- `cse_ova_file`: This will reference the file name of the CSE OVA, like `VMware_Cloud_Director_Container_Service_Extension-4.0.1.ova`.
+
+-> To download the required OVAs, please refer to the [CSE documentation][cse_docs].
+
+~> Both CSE Server and TKGm OVAs are heavy. Please take into account that the upload process could take more than 30 minutes, depending
+on upload speed. You can tune the `upload_piece_size` to speed up the upload. Another option would be uploading them manually in the UI and
+using the [vcd_catalog_vapp_template][catalog_vapp_template_ds] data source instead.
+
+If you need to upload more than one OVA, please modify the [proposed configuration][step2], or if you want to use existing OVAs you can also leverage
+using the [vcd_catalog_vapp_template][catalog_vapp_template_ds] data source instead.
+
+### "Kubernetes Cluster Author" global role
+
+Apart from the role to administrate the CSE Server created in [step 1][step1], we also need a [Global Role][global_role]
```

Comment:
"administrate" sounds [interesting](https://english.stackexchange.com/questions/116384/administer-or-administrate) here :) I was thinking about "administer", but probably even easier is "manage":

```suggestion
Apart from the role to manage the CSE Server created in [step 1][step1], we also need a [Global Role][global_role]
```

Reply:
My "Spanglish" attacked here :)
Changes!